app/__init__.py

```diff
@@ -100,19 +100,18 @@ def create_app():
         'version': 1,
         'handlers': {
             'console': {
-                'level': logging.WARN,
                 'class': 'logging.StreamHandler',
                 'stream': 'ext://sys.stdout'
             }
         },
         'loggers': {
             '': {
                 'handlers': ['console'],
-                'level': logging.WARN
+                'level': logging.INFO
             },
             'default': {
                 'handlers': ['console'],
-                'level': logging.INFO
+                'level': logging.WARN
```

Comment:
Are requests being logged now?

And only INFO from our application gets printed?

Reply:
I think opposite will happen. INFO from libraries will get print and from our app won't

Reply:
@iamareebjamal Yes, requests are being logged.